torchgeo/trainers/byol.py

```diff
@@ -261,7 +261,7 @@ def __init__(
             image_size: the size of the training images
             hidden_layer: the hidden layer in ``model`` to attach the projection
                 head to, can be the name of the layer or index of the layer
-            input_channels: number of input channels to the model
+            in_channels: number of input channels to the model
```

Comment:
This trainer was pretty out-of-sync with our other trainers and uses different key names. Tried to sync them a bit more.